articles/protocols/saml/idp-initiated-sso.md

```diff
@@ -13,9 +13,7 @@ useCase:
 
 # IdP-Initiated SSO
 
-**Beginning with auth0.js v9.3.4, you must [enable the impersonation flags](/user-profile/user-impersonation#enable-impersonation) to use IdP-initiated login.**
```

Comment:
Is this statement not true / no longer true?

Reply:
Oh, I follow, you've just added that entire section here, so no need to link anymore

Reply:
Yeah... and it looks about as bad as I imagined... I'll add a heading and move this down below to make this page less ugly.

Reply:
@jeffreylees Ok, I think this looks better now. Feel free to carry on with the review process.